src/common/converters/tests/data_decoded.rs

```diff
@@ -899,3 +900,11 @@ fn nested_delegate_in_multi_send_without_nested_delegate() {
 
     assert_eq!(data_decoded.has_nested_delegated(), false);
 }
+
+#[test]
+fn no_param_data_decoded_has_nested_delegate() {
+    let data_decoded =
+        serde_json::from_value::<DataDecoded>(json!({"method":"parameterlessMethod"})).unwrap();
```

Comment:
Do we have a test for a call with nested delegate calls that is not a multisend?

Reply:
not really 🤔 I will add one 👍

Reply:
I am not able to find a suitable example for this. Generally, `execTransaction` comes to mind but I can't seem to find an example that contains `InternalTransaction`s. 

Currently there aren't any checks on the name of the method. I assume you want this test to assert that we don't check only `multiSend`, right ?